pkg/tests/observability_default_config_test.go

```diff
@@ -34,7 +34,7 @@ var _ = Describe("Observability:", func() {
 		}
 		observabilityAddonSpec := mcoRes.Object["spec"].(map[string]interface{})["observabilityAddonSpec"].(map[string]interface{})
 		Expect(observabilityAddonSpec["enableMetrics"]).To(Equal(true))
-		Expect(observabilityAddonSpec["interval"]).To(Equal(int64(30)))
+		Expect(observabilityAddonSpec["interval"]).To(Equal(int64(60)))
```

Comment:
The value is from MCO CR in https://github.com/open-cluster-management/observability-gitops/blob/ac83f3787f1574f568d4665f2a2a111f85ea720b/mco/e2e/v1beta2/observability.yaml#L12. I am not sure if you want to change as well. I think we may just leave as it is - the customized value is 30s.

Reply:
We're using `interval: 30` for e2e cases. So we don't need to change this.
We may later need to update this to get the interval from MCO CR.

Reply:
@morvencao 
thanks, close this pr